FeedStoreChallenge/RealmStore.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  RealmStore.swift
+//  FeedStoreChallenge
+//
+//  Created by Пильтенко Роман on 13.02.2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import RealmSwift
+
+public final class RealmStore: FeedStore {
+	
+	private let configuration: Realm.Configuration
+	
+	public init(fileURL: URL) {
+		self.configuration = Realm.Configuration(fileURL: fileURL, schemaVersion: 1, migrationBlock: { _, oldSchemaVersion in
+			if (oldSchemaVersion < 1) {
+			  // Nothing to do!
+			  // Realm will automatically detect new properties and removed properties
+			  // And will update the schema on disk automatically
+			}
+		})
+	}
+	
+	public func deleteCachedFeed(completion: @escaping DeletionCompletion) {
+		do {
+			let database = try Realm(configuration: configuration)
+			try database.write {
+				database.deleteAll()
+				completion(nil)
+			}
+		} catch let error as NSError {
+			completion(error)
+		}
```

Comment:
The casting is unncessary:

```suggestion
		} catch {
			completion(error)
		}
```